neurons/calculator/calculator.py

```diff
@@ -0,0 +1,81 @@
+# encoding=utf8  
+import math
+
+from core.NeuronModule import NeuronModule, InvalidParameterException
+from core import Utils
+
+
+class Calculator(NeuronModule):
+    def __init__(self, **kwargs):
+
+        super(Calculator, self).__init__(**kwargs)
+
+        """
+        Ideally, a parser, lexer, tokenizer would be the best option for a
+        calculator app. But this works ok for simple operations...
+        """
+
+        # get operation from brain
+        self.operation  = kwargs.get('operation', None)
+
+        # 3 following lines force STT to math operators
+        self.operation  = self.operation.replace('divisé par', '/')
```

Comment:
This part is also only concerned by french language only. We really want that each kalliope's neuron can handle multi lang